src/rebar_utils.erl

```diff
@@ -70,7 +70,9 @@
          info_useless/2,
          list_dir/1,
          user_agent/0,
-         reread_config/1]).
+         reread_config/1,
+         get_proxy_auth/0,
+         set_proxy_auth/1]).
```

Comment:
Am I right in understanding that this function is never called except through `set_http_options` outside of tests?

What would you think of not exposing it in this interface? This may require to modify the tests though.